tests/Browser/console/Tests_Browser_LoginTest_testPathWrong-0.log

```diff
@@ -0,0 +1,8 @@
+[
+    {
+        "level": "SEVERE",
+        "message": "http:\/\/localhost:8000\/abc - Failed to load resource: the server responded with a status of 404 (Not Found)",
+        "source": "network",
+        "timestamp": 1580785991118
+    }
+]
```

Comment:
Please add blank line and the end of file.

Reply:
it's file log had created when run test... can i fixe it?.

Reply:
If this file created after run each test, i think we don't need to add this file to source code.
Because:
+ As i guess content of this file will change with every run test("timestamp": 1580785991118 is 10h AM 04/02/2020).
+ This content is not meant to save(we can create every run test and only use for that check time).
Please remove it.

Reply:
i removed it